File: 30-Days-of-Code/Day-1-Data-Types/PHP/solution.php
<?php
$handle = fopen ("php://stdin","r");
$i = 4;
$d = 4.0;
$s = "HackerRank ";

// Declare second integer, double, and String variables.
$j;
$e;
$t;

// Read and save an integer, double, and String to your variables.
$j = fgets($handle);
$e = fgets($handle);
$t = fgets($handle);

// Print the sum of both integer variables on a new line.
printf("%d\n", $i+$j);

// Print the sum of the double variables on a new line.
printf("%.1f\n", $d+$e);

// Concatenate and print the String variables on a new line
// The 's' variable above should be printed first.
print($s.$t);

fclose($handle);
?>

File: 30-Days-of-Code/Day-1-Data-Types/Scala/solution.scala
object Solution {
    def main(args: Array[String]) {
        val i = 4
        val d = 4.0
        val s = "HackerRank "
        
        // Read values of another integer, double, and string variables
        // Note use scala.io.StdIn to read from stdin
        val j = scala.io.StdIn.readInt()
        val e = scala.io.StdIn.readDouble()
        val t = scala.io.StdIn.readLine()
        
        // Print the sum of both the integer variables
        println(i+j)
        
        // Print the sum of both the double variables
        println(d+e)
        
        // Concatenate and print the string variables
        // The 's' variable above should be printed first.
        println(s+t)
    }
}
